src/lib/components/stepper/components/transact-step.svelte

```diff
@@ -318,6 +325,121 @@
           message: 'Waiting for you to confirm in your wallet',
         });
 
+        if (executingTx.gasless) {
+          const domain = {
+            name: 'Caller',
+            version: '1',
+            chainId: network.chainId,
+            verifyingContract: networkConfig.contracts.CALLER,
+          };
+
+          const types = {
+            CallSigned: [
+              {
+                name: 'sender',
+                type: 'address',
+              },
+              {
+                name: 'target',
+                type: 'address',
+              },
+              {
+                name: 'data',
+                type: 'bytes',
+              },
+              {
+                name: 'value',
+                type: 'uint256',
+              },
+              {
+                name: 'nonce',
+                type: 'uint256',
+              },
+              {
+                name: 'deadline',
+                type: 'uint256',
+              },
+            ],
+          };
+
+          const { data, to } = executingTx.transaction;
+          assert(data, 'Expected data to be defined');
+
+          const { provider } = get(walletStore);
+          const caller = new Contract(networkConfig.contracts.CALLER, callerAbi, provider);
+
+          const nonce = await caller.nonce(address);
+
+          const payload = {
+            sender: address,
+            target: to,
+            data,
+            value: 0,
+            nonce: Number(nonce),
+            // 1 hour in seconds
+            deadline: Math.floor(Date.now() / 1000) + 3600,
+          };
+
+          const sig = await signer.signTypedData(domain, types, payload);
+
+          updateTransactionTimelineStatus(executingTx, {
+            status: 'pending',
+            message: 'Submitting transaction',
+          });
+
+          const gaslessCallRes = await fetch('/api/gasless/call', {
```

Comment:
Now that I have `fetch` on the brain and I see that you check the res below, should you check for ok here as well?

Reply:
yes, prob best to check explicitly 👍 will add that